tests/test_sync.py

```diff
@@ -103,3 +104,34 @@ def test_is_changed(self):
             "updated_at": "2020-08-31T22:31:55"
         }
         self.assertFalse(gazu.sync.is_changed(source_asset, target_asset))
+
+    def test_get_last_events(self):
+        result = [{
+            "id": fakeid("event-1"),
+            "created_at": "date-1",
+            "name": "name-1",
+            "user_id": fakeid("user-1"),
+            "data": {
+                "project_id": fakeid('project-1')
+                }
+            }
+            ]
+        path = "/data/events/last"
+        with requests_mock.mock() as mock:
+            mock.get(gazu.client.get_full_url(path), text=json.dumps(result))
+            self.assertEqual(gazu.sync.get_last_events(), result)
+            self.assertEqual(gazu.sync.get_last_events(project=fakeid('project-1'), after="2021-11-06", before="2021-11-06"), result)
+
+    def test_validate_date_format(self):
```

Comment:
It should be in the  `test_helpers` module.

Reply:
It was alread done by this commit https://github.com/cgwire/gazu/pull/190/commits/5bb2982b1ca293def3bb100f4c2a893241672cf5